projects/js-toolkit/packages/liferay-cli/src/new/index.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: LGPL-3.0-or-later
+ */
+
+import {format} from '@liferay/js-toolkit-core';
+import path from 'path';
+
+import * as targetLiferay from './target-liferay';
+import HumanError from './util/HumanError';
+
+const {error, info, print, success, text, title} = format;
+
+/* eslint-disable-next-line @typescript-eslint/no-explicit-any */
+export type OptionValue = any;
+
+export interface Options {
+	[key: string]: OptionValue;
+}
+
+interface Facet {
+	processOptions(options: Options): Promise<Options>;
+	render(options: Options): Promise<void>;
+}
+
+interface Target extends Facet {
+	name: string;
+}
+
+export default async function newProject(
+	name: string,
+	batch?: boolean
+): Promise<void> {
+	/* eslint-disable-next-line @typescript-eslint/no-var-requires */
+	const {version} = require('../../package.json');
+
+	print(title`|👋 |Welcome to Liferay Project Generator v${version}`);
+
+	const target: Target = targetLiferay;
```

Comment:
Is this type not inferred?

Reply:
I guess it should, but I added it because my "IDE" (vim+CoC) was not recognizing it. However, looking at the code now, it seems like it should have recognized it :shrug: . I will check again when I prepare the chunk PR for this.